android/app/src/main/java/io/coursepick/coursepick/presentation/filter/RoundedCornerToggleButton.kt

```diff
@@ -0,0 +1,64 @@
+package io.coursepick.coursepick.presentation.filter
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.selection.toggleable
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.PreviewLightDark
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import io.coursepick.coursepick.R
+import io.coursepick.coursepick.presentation.search.ui.theme.CoursePickTheme
+
+@Composable
+fun RoundedCornerToggleButton(
+    label: String,
+    isActive: Boolean,
+    onActiveChanged: () -> Unit,
+    modifier: Modifier = Modifier,
+    enabled: Boolean = true,
+) {
+    Box(
+        modifier =
+            modifier
+                .toggleable(
+                    value = isActive,
+                    enabled = enabled,
+                    role = Role.Checkbox,
+                    onValueChange = { onActiveChanged() },
+                ).clip(RoundedCornerShape(size = 8.dp))
+                .background(if (isActive) colorResource(R.color.point_secondary) else colorResource(R.color.gray2))
+                .padding(vertical = 20.dp)
+                .padding(horizontal = 4.dp),
```

Comment:
한 번만 호출할 수 있지 않을까요? 👀 

```kotlin
@Stable
public fun Modifier.padding(
    horizontal: Dp = 0.dp,
    vertical: Dp = 0.dp
): Modifier
```
```suggestion
                .padding(horizontal = 4.dp, vertical = 20.dp),
```